subsys/bluetooth/host/smp.c

```diff
@@ -2111,6 +2138,27 @@ static u8_t send_pairing_rsp(struct bt_smp *smp)
 }
 #endif /* CONFIG_BT_PERIPHERAL */
 
+static u8_t smp_pairing_allow_query(
+	struct bt_conn *con,
```

Comment:
Use `conn` for consistency with existing code. Also, move it to the first line since you should only be splitting when you otherwise go beyond 80 characters.